integrations/haproxy/documentation.yaml

```diff
@@ -1,27 +1,34 @@
-exporter_type: sidecar
+exporter_type: included
 app_name_short: HAProxy
 app_name: {{app_name_short}}
 app_site_name: HAProxy
 app_site_url: https://www.haproxy.com/
-exporter_name: the HAProxy exporter
-exporter_pkg_name: haproxy_exporter
-exporter_repo_url: https://github.com/prometheus/haproxy_exporter
+exporter_name: the HAProxy promex exporter
+exporter_pkg_name: haproxy_promex
+exporter_repo_url: https://github.com/haproxy/haproxy/blob/master/addons/promex/README
 dashboard_available: true
-minimum_exporter_version: v0.13.0
+minimum_exporter_version: "2.4"
 multiple_dashboards: false
 dashboard_display_name: {{app_name_short}} Prometheus Overview
-config_mods: |
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics only when it is 
+  [built with the service enabled](https://github.com/haproxy/haproxy/blob/master/addons/promex/README) and an appropriate `frontend` is included in the configuration.
+  Most of the official docker images for versions greater than or equal to 2.4 are built with this service enabled.
+  The following example was built referencing [HAProxy Enterprise documentation](https://www.haproxy.com/documentation/hapee/latest/observability/metrics/prometheus/).
+  It works with the community edition and can be modified to suit specific needs:
+  <pre>
```

Comment:
Is this intended to be included inline (and not extracted into the an external file (which is what we do with "config_mods" As is, this will just be an inline block.

Reply:
Hi, Austin! Hope you're doing well, too! 

I don't have a strong opinion about this -- I realize that you don't have great insight into what we do with this downstream, so I just wanted to make sure that "inline" reflected your intent. So I have no objection. If you and YQ are good with it, that's good enough for me!

Reply:
I'm good with this, if everyone else is.